<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabMain">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tabFiles">
        <attribute name="title">
         <string>Files</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QWidget" name="wTop" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="BCPathBar" name="framePathBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wMiddle" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitterFiles">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QStackedWidget" name="stackFiles">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="pageTvFiles">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QTreeView" name="treeViewFiles">
                   <property name="alternatingRowColors">
                    <bool>true</bool>
                   </property>
                   <property name="selectionMode">
                    <enum>QAbstractItemView::MultiSelection</enum>
                   </property>
                   <property name="uniformRowHeights">
                    <bool>true</bool>
                   </property>
                   <property name="sortingEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pageLvThumbnails">
                <widget class="QLabel" name="label_5">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>50</y>
                   <width>89</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Test thumb list</string>
                 </property>
                </widget>
               </widget>
              </widget>
              <widget class="QTabWidget" name="tabFilesDetails">
               <property name="tabShape">
                <enum>QTabWidget::Rounded</enum>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <property name="documentMode">
                <bool>false</bool>
               </property>
               <property name="movable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="tabFilesDetailsInformations">
                <attribute name="title">
                 <string>Informations</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>6</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <property name="bottomMargin">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QFrame" name="frameFileInformation">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <widget class="QLabel" name="label_3">
                    <property name="geometry">
                     <rect>
                      <x>40</x>
                      <y>40</y>
                      <width>191</width>
                      <height>24</height>
                     </rect>
                    </property>
                    <property name="text">
                     <string>Test preview</string>
                    </property>
                   </widget>
                  </widget>
                 </item>
                 <item>
                  <widget class="Line" name="line_3">
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <property name="lineWidth">
                    <number>1</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widgetFilePreview" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QToolBox" name="toolBoxInfo">
                      <property name="frameShape">
                       <enum>QFrame::NoFrame</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Plain</enum>
                      </property>
                      <property name="lineWidth">
                       <number>1</number>
                      </property>
                      <property name="midLineWidth">
                       <number>0</number>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <widget class="QWidget" name="pageFileNfoGeneric">
                       <property name="geometry">
                        <rect>
                         <x>0</x>
                         <y>0</y>
                         <width>536</width>
                         <height>237</height>
                        </rect>
                       </property>
                       <attribute name="label">
                        <string>Information</string>
                       </attribute>
                      </widget>
                      <widget class="QWidget" name="pageFileNfoKra">
                       <property name="geometry">
                        <rect>
                         <x>0</x>
                         <y>0</y>
                         <width>536</width>
                         <height>199</height>
                        </rect>
                       </property>
                       <attribute name="label">
                        <string>Krita file information</string>
                       </attribute>
                      </widget>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tabFilesDetailsDirTree">
                <attribute name="title">
                 <string>Directory tree</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QTreeWidget" name="treewDirectoryTree">
                   <column>
                    <property name="text">
                     <string notr="true">1</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wBottom" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="lblFileNfo">
              <property name="font">
               <font>
                <family>DejaVu Sans Mono</family>
                <pointsize>9</pointsize>
                <italic>false</italic>
               </font>
              </property>
              <property name="toolTip">
               <string>Number of [selected] files, Size of [selected] files</string>
              </property>
              <property name="statusTip">
               <string>Number of [selected] files, Size of [selected] files</string>
              </property>
              <property name="whatsThis">
               <string>Number of [selected] files, Size of [selected] files</string>
              </property>
              <property name="text">
               <string>0 Files, 0.00KiB</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblDiskNfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>DejaVu Sans Mono</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Free space available on disk</string>
              </property>
              <property name="statusTip">
               <string>Free space available on disk</string>
              </property>
              <property name="whatsThis">
               <string>Free space available on disk</string>
              </property>
              <property name="text">
               <string>0.00MiB free</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btTabFilesLayoutModel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>48</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Reset default layout</string>
              </property>
              <property name="statusTip">
               <string>Reset default layout</string>
              </property>
              <property name="whatsThis">
               <string>Reset default layout</string>
              </property>
              <property name="text">
               <string>l</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::MenuButtonPopup</enum>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonIconOnly</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::NoArrow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDocuments">
        <attribute name="title">
         <string>Documents</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BCPathBar</class>
   <extends>QFrame</extends>
   <header>bulicommander.bc.bcpathbar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
